### ISSUE TITLE
send photo/video to Telegram messenger instead of emailing

### ISSUE THREAD

**Issue by mrmate2** [mtnbkr88/ESP32CAMRemotePIR#2]
Dear author, is it difficuilt for  You to apply such functionality: send photo/video to Telegram messenger instead of emailing or parallel with emailing?
Many thanks in advance!

**Comment by mtnbkr88** [comment]
I don't know what a Telegram messenger is, but if its something that can be reached through the internet, then I expect it can be done. Aside from the ESP32CAMs often failing to start the camera when its hot outside, I've got these things working good enough to meet my needs and I've moved on to other things. You are free to download code and add this functionality if you want. It would be a good learning experience for you. That's how I learned. I started with something that was close to what I wanted. I figured out how it worked, then I modified it to do what I wanted.

Good Luck and have fun. These ESP32CAMs are for fun and entertainment. They are too flaky to be used for anything critical.
